packages/components/src/components/Table/TableContainer.tsx

```diff
@@ -0,0 +1,18 @@
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { Card } from '../Card/Card';
+
+export const Container = styled.div`
+    flex-direction: column;
+`;
+
+interface TableContainerProps {
+    children: ReactNode;
+}
+
+export const TableContainer = ({ children }: TableContainerProps) => (
```

Comment:
Why not just Table?

Reply:
To me table is the whole components like TableRow and other components together, this is just a container. But don't have strong opinion on this.

Reply:
Then please rename it to Table 🙏  we might change it in the future if it will not fit.